dev/TEAL_opcodes.md

```diff
@@ -414,6 +416,7 @@ FirstValidTime causes the program to fail. The field is reserved for future use.
 | 3 | ZeroAddress | []byte | 32 byte address of all zero bytes |
 | 4 | GroupSize | uint64 | Number of transactions in this atomic transaction group. At least 1. |
 | 5 | LogicSigVersion | uint64 |  |
+| 6 | Round | uint64 | Current round number |
```

Comment:
can we bring back real time while we're at it? If not the current-block according to the proposer, then the time of the previous block. Just for stateful-eval.